src/Github.hs

```diff
@@ -76,7 +76,8 @@ data PullRequestPayload = PullRequestPayload {
   branch      :: Branch,   -- Corresponds to "pull_request.head.ref".
   sha         :: Sha,      -- Corresponds to "pull_request.head.sha".
   title       :: Text,     -- Corresponds to "pull_request.title".
-  author      :: Username  -- Corresponds to "pull_request.user.login".
+  author      :: Username, -- Corresponds to "pull_request.user.login".
+  body        :: Text      -- Corresponds to "pull_request.body"
```

Comment:
Can you add a newtype for Text, it conveyes a bit more meaning. Might also be best to stick to Github's API spec, with the `body` being something like `Maybe Body`.